### PR TITLE
Update docs explaining the stale bot usage

### PR DESCRIPTION
**Pull request by manishgarg1**
#### Description of changes

Updated the documentation with stale bot usage and some other minor fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,14 +48,11 @@ ReactDOM.render(
   document.body.firstChild
 );
 ```
-**[⬆ back to top](#Contents)**
```
Comment (jordandrako) [inline README.md]: Why did you remove all the links to the ToC?

Comment (manishgarg1) [reply README.md]: They were not working on GitHub. Will figure later why but for now they were creating noise and no value.

---
In reply to: [180925584](https://github.com/OfficeDev/office-ui-fabric-react/pull/4523#discussion_r180925584) [](ancestors = 180925584)

File: FAQ.md [hunk 2]
```diff
@@ -126,4 +126,22 @@ We try to fix high priority bugs ASAP.
...
+
+### But my issue is relevant, it’s just a long discussion! What can I do?
+
+Stale-bot is versatile and has exemption labels, projects, and milestones. As of right now any issues marked 'Priority 0', 'Priority 1', 'Needs: discussion', and 'Needs: revisit' will never be marked stale.
+
+We ask everyone to not abuse these. This cleanup is for better organization of the Fabric React project with the goal to increase visibility and response time on bugs and feature requests and organizing the issues is key to this endeavor. If there is a more semantic label to express, why your issue needs to stick around for multiple months please contact Michael Angotti (v-micang@microsoft.com) so he can factor that into the organization.
```
[the remark this answers is not in the document]
Comment (manishgarg1) [reply FAQ.md]: Removed

---
In reply to: [180925951](https://github.com/OfficeDev/office-ui-fabric-react/pull/4523#discussion_r180925951) [](ancestors = 180925951)

File: README.md (hunk 1 above)
Comment (oengusmacinog) [reply README.md]: I was assuming @manishgarg1 decided against them.  Looked like he added them the other day.

Comment (manishgarg1) [reply README.md]: No, they were just not working. So I am yanking them out. Will fix them later in a way that they are funcitonal :)

---
In reply to: [180927395](https://github.com/OfficeDev/office-ui-fabric-react/pull/4523#discussion_r180927395) [](ancestors = 180927395)
